packages/labextension/schema/plugin.json

```diff
@@ -45,6 +45,12 @@
       "title": "Only kernel whitelist",
       "description": "Show only environment corresponding to whitelisted kernels",
       "default": false
+    },
+    "condaStoreUrl": {
```

Comment:
Question: should the settings be shared between the plugins?

Reply:
I could be convinced to either move this setting off on its own or to keep it here. From a user perspective, I think I would want to only go to one place to change environment management settings, but I'll leave the final decision up to you.

Reply:
Let's keep them together then - we can revise that later if the number of settings grows a lot.